scripts/credstashCli.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Created by Idan on 6/14/18
+ */
+
+const process = require('process');
+const program = require('commander');
+const _ = require('lodash');
+const CredstashApi = require('../lib/credstashApi');
+
+function getValue(table, appKey = undefined, key = undefined) {
+    this.credStashApi = new CredstashApi({table: table});
+    let storedValue = null;
+
+    if (appKey === undefined) {
+        storedValue = this.credStashApi.getAll();
+        Object.keys(storedValue).forEach(
+            function (keyName) {
```

Comment:
maybe defining this function with a **meaningful name** on this file and then running 
_.each(_.keys(storedValue), functionWIthMeaningfulName)
u will get a more simple getter and documentation for this logic

Reply:
done